### DISCUSSION TITLE
MSMTP integration with others containers

### DISCUSSION THREAD

**Discussion by lunitaris** [crazy-max/docker-msmtpd#63] [Q&A]
Hello,

I am using podman, to build a Hagrid server.
I am using https://github.com/chimbosonic/hagrid-container.

For info, docker or podman, whaterever, commands are king of the same.

**container1 = hagrid server,** it uses the 'sendmail' command by default.
**container2=msmtp server** (will act as a SMTP relay, sending mails through a gmail account.

How do I tell another container to use the SMTP service of the msmtp container?
Through a socket? How?

<img width="1096" alt="image" src="https://github.com/crazy-max/docker-msmtpd/assets/11634917/c74f7640-3019-4e58-80d1-24bb45cf25e9">

**Comment by mmallejac** [comment]
I am also using the same kind of setup.

Simply configure MSMTPD so that it can send email, and from your container1, just send your mail to MSMTP, using its container name, say `msmtp` and the port defined in your docker compose file (or the -p option used to start msmtp), usually 2500.

Not sure about Hagrid configuration, but somewhere specify `msmtp:2500` 

Both containers have to be on the same network, most probably on your `Front-end` one, based on your drawing.
Hope this helps !
